scripts/setup_irods.py

```diff
@@ -459,9 +462,10 @@ def setup_client_environment(irods_config):
 
     print('\n', end='')
 
+    # TODO Document changing the schema version here when any of the json schemas are changed.
     service_account_dict = {
             'schema_name': 'service_account_environment',
-            'schema_version': 'v4',
+            'schema_version': 'v5',
```

Comment:
Handle TODO.

This should be captured in the development docs of irods/irods_docs. It needs to explain why it is changed here rather than upgrade_configuration.py. Consider how we handle server_config.json. Why can't the service account JSON file have something similar?

Reply:
Created #8000

Reply:
Will be addressed in a separate PR.